app/views/statistics/index.html.erb

```diff
@@ -1,6 +1,10 @@
 <% title _("Statistics"), _('Statistics') %>
 <div id="statistics">
-<% charts.each do |ch| %>
-  <%= content_tag(:div, ch, :class=>"stats-well") %>
-<% end %>
+  <% @stats.each do |stat| %>
+    <div data-ajax-url=<%= stat.url %> class='stats-well' data-on-complete='refreshCharts'>
+      <div class='statistics-pie-placeholder'>
+        <h4 class='ca pie-title'> <span class='spinner spinner-xl spinner-inline'> </span> <%= stat.title %> </h4>
```

Comment:
There is no `spinner-xl`, the largest is `spinner-lg`. Also, looks strange to have the spinner inline before the title, maybe move it down to where the chart should be?